src/vuestic-theme/vuestic-sass/overrides/_custom-libs.scss

```diff
@@ -112,10 +112,10 @@
   }
 }
 
-//v-tooltip
+//v-tooltip - check this one please, looks like it does nothing, though not entirely sure
```

Comment:
You can check tooltip page. If everything looks fine (compare with stage), removal is also fine.